admin-ui/app/routes/Apps/Gluu/GluuSelectRow.tsx

```diff
@@ -56,48 +56,71 @@ function GluuSelectRow({
     })
   }
 
+  // Calculate grid sizes based on Bootstrap column system (out of 12)
+  const labelGridSize = Math.round((lsize / 12) * 12)
+  const selectGridSize = Math.round((rsize / 12) * 12)
+
   return (
-    <FormGroup row>
-      <GluuLabel
-        label={label}
-        size={lsize}
-        doc_category={doc_category}
-        doc_entry={doc_entry || name}
-        required={required}
-      />
-      <Col sm={rsize}>
-        <InputGroup>
-          <CustomInput
-            type="select"
+    <Grid container spacing={2} alignItems="center" sx={{ mb: 2 }}>
+      <Grid item xs={labelGridSize}>
+        <GluuLabel
+          label={label}
+          size={lsize}
+          doc_category={doc_category}
+          doc_entry={doc_entry || name}
+          required={required}
+        />
+      </Grid>
+      <Grid item xs={selectGridSize}>
+        <FormControl fullWidth error={showError} disabled={disabled}>
+          <Select
+            labelId={`${name}-label`}
             id={name}
             data-testid={name}
+            size="small"
             name={name}
-            defaultValue={value}
-            onChange={(event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
+            value={value || ''}
+            onChange={(event) => {
```

Comment:
Manually creating a synthetic event object inside the `onChange` handler is bad because it adds unnecessary complexity and boilerplate code; the original event should be used directly where possible.